### DISCUSSION TITLE
Loco Poisitoning on VM

### DISCUSSION THREAD

**Discussion by matosant** [bitcraze/discussions#1036] [Q&A]
Hello folks, 

I've been having trouble connecting my nodes on the VM provided by crazy fly it won't allow me to select anything on the interface even though the VM is recognizing that the USB and radio are plugged in and scanned. I performed all the steps required to flash and apply the numbers to the nodes I just don't have an idea why the VM won't work. hopefully someone can help, thanks.

**Comment by ataffanel** [comment]
I will need a bit more information to be able to help:
 - What is the node state, is it in DFU or firmware mode
 - Did you install any drivers on your host (Windows?)
 - What is the lps tool showing in the VM, are you able to make it show the flashing/bootloader screen when plugging your node in bootloader mode?

Any other details about your setup and the failure would help.

**Reply by matosant** [reply]
we tried to connect it through both states, if what you're asking about is the drivers we installed Zadig, and the LPS configuration we did all of those steps that had to do with the radio and anchors. I am not sure what you're asking for the last part but the VM still won't work, i went onto the bootloader mode and downloaded the firmware for the crazy radio but I have no idea what to do next.

**Reply by knmcguire** [reply]
we are currently in the process to investigate this. In the mean time, we found out if the LPS tool is installed from source (with python) and not with .exe, it seems to work. While we fix the windows executable you can try that back on windows again. See the readme here https://github.com/bitcraze/lps-tools
